XMonad/Hooks/StatusBar.hs

```diff
@@ -124,12 +126,12 @@ import XMonad.Hooks.StatusBar.PP
 --
 -- If your status bar does not support property-based logging, you may also try
 -- 'statusBarPipe'.
--- It can be used in the same way as 'statusBarProp' above (for xmobar, you will now
--- use the @StdinReader@ plugin in your @.xmobarrc@).  Instead of writing to
--- a property, this function opens a pipe and makes the given status bar read
--- from that pipe.
+-- It can be used in the same way as 'statusBarProp' above (for xmobar, you now
+-- have to use the @StdinReader@ plugin in your @.xmobarrc@).  Instead of
```

Comment:
Since StdinReader is default in xmobar, I'm not sure the strong "have to" wording here is appropriate.

Reply:
Seems appropriate to me, since using this without either `StdinReader` or `UnsafeStdinReader` will lead to a hang, which is why all this rewiring.

Reply:
It is the default, but that's void as soon as people start customizing their `template` string (which I suspect most will have done).  

Really, this part is written for people who may have tried the property-logging solution above but want to switch to pipes again for one reason or another.  Requiring people to read every little bit is perhaps not the best way to go about things, but it's quite hard to find a wording that satisfies both sides here I feel.

Reply:
Fair enough :-)